ConsoleView.h

```diff
@@ -1,52 +1,89 @@
-#ifndef UML_CONSOLE_VIEW
-#define UML_CONSOLE_VIEW
-#include <string>
-#include "UMLAttribute.h"
-#include "UMLClass.h"
+#include "ConsoleView.h"
 #include "UMLRelationship.h"
-#include <vector>
-#include <list>
 #include <iostream>
-#include <algorithm>
+//All these functions return void, they are simply here 
+//for the user to call when they want to get information on 
+//a CLASS (or a list of all classes), RELATIONSHIP, or ATTRIBUTE
 
-/*
-    The ConsoleView is responsible for displaying information to the user.
-    Very little if any application logic is handled in the view class.
-*/
-class ConsoleView
-{
+ConsoleView::ConsoleView(std::string name){
+	this->Name = name;
+}
 
-public:
-    ConsoleView(){};
-  
+std::ostream& operator<<(std::ostream& out, const UMLRelationship& dt) {
```

Comment:
- Why are we suddenly implementing methods in the header file? The compiler won't like this at all. The header file is for your forward declarations and the cpp file is for your implementation.